.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Storybook
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🏷️
+        uses: actions/checkout@v2
+
+      - name: Match tag to package version 🧐
+        uses: geritol/match-tag-to-package-version@0.0.2
+        env:
+          TAG_PREFIX: refs/tags/v
+
+      - name: Expose version as env var 🤓
+        uses: nyaayaya/package-version@v1
+
+      - name: Use Node 12 🕹️
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12'
+
+      - name: Install ⚙️
+        uses: bahmutov/npm-install@v1
+        with:
+          working-directory: packages
+
+      - name: Build 🛠️
+        run: cd packages && npm run build
+
+      - name: Set @h5web/app version 📌
+        uses: reedyuk/npm-version@vlatest
+        with:
+          version: ${{ env.PACKAGE_VERSION }}
```

Comment:
Can't we instead use the tag ? Just to avoid the _expose as env var_ action.

Reply:
The name of the tag can be parsed from env var `GITHUB_REF`, but it's something like `refs/tags/vX.Y.Z`, so it's not as trivial as using an action.